src/app/app.component.html

```diff
@@ -1 +1,213 @@
+                  {{ item.name }}
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="col-12 col-md-12 col-lg-4">
+        <div class="row g-4">
+          <div class="col-12">
+            <h6 class="mb-4">Get to know us</h6>
+            <ul class="nav flex-column">
+              <li class="nav-item py-1">
+                <a [routerLink]="'/'" class="nav-link">
+                  Company
+                </a>
+              </li>
+              <li class="nav-item py-1">
+                <a [routerLink]="'/'" class="nav-link">
+                  About
+                </a>
+              </li>
+              <li class="nav-item py-1">
+                <a [routerLink]="'/'" class="nav-link">
+                  Blog
+                </a>
+              </li>
+              <li class="nav-item py-1">
+                <a [routerLink]="'/'" class="nav-link">
+                  Help Center
+                </a>
+              </li>
+              <li class="nav-item py-1">
+                <a [routerLink]="'/'" class="nav-link">
+                  Our Value
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="border-top py-4">
+      <div class="row align-items-center">
+        <div class="col-lg-5 text-lg-start text-center mb-2 mb-lg-0">
+          <ul class="list-inline mb-0">
+            <li class="list-inline-item text-dark">Payment Partners</li>
+            <li class="list-inline-item">
+              <a [routerLink]="'/'">
+                <img src="../assets/images/payment/amazonpay.svg" alt="">
+              </a>
+            </li>
+            <li class="list-inline-item">
+              <a [routerLink]="'/'">
+                <img src="../assets/images/payment/american-express.svg" alt="">
+              </a>
+            </li>
+            <li class="list-inline-item">
+              <a [routerLink]="'/'">
+                <img src="../assets/images/payment/mastercard.svg" alt="">
+              </a>
+            </li>
+            <li class="list-inline-item">
+              <a [routerLink]="'/'">
+                <img src="../assets/images/payment/paypal.svg" alt="">
+              </a>
+            </li>
+            <li class="list-inline-item">
+              <a [routerLink]="'/'">
+                <img src="../assets/images/payment/visa.svg" alt="">
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="col-lg-7 mt-4 mt-md-0">
+          <ul class="list-inline mb-0 text-lg-end text-center">
+            <li class="list-inline-item mb-2 mb-md-0 text-dark">Get deliveries with FreshCart</li>
+            <li class="list-inline-item ms-4">
+              <a [routerLink]="'/'">
+                <img src="../assets/images/appbutton/appstore-btn.svg" alt="" style="width: 140px;">
```

Comment:
use absolute path in whole PR

Reply:
do not use inline styles

Reply:
I am not sure if I understand what should I do. Do you mean that I should use routerLink="/" - without square brackets? I found on stackoverflow that without brackets you're passing string only and you can't change it, it's hard coded and it'll be like that throughout your app. ([found here](https://stackoverflow.com/questions/41370760/difference-between-routerlink-and-routerlink))